fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -188,19 +190,20 @@ public boolean canCreate(String resourceType, Object resource) {
                       try {
                         fiatService.canCreate(username, resourceType, resource);
                         return true;
-                      } catch (RetrofitError re) {
+                      } catch (NotFoundException e) {
+                        return false;
+                      } catch (SpinnakerHttpException e) {
                         boolean shouldRetry = true;
-                        if (re.getKind() == RetrofitError.Kind.HTTP) {
-                          switch (HttpStatus.valueOf(re.getResponse().getStatus())) {
-                            case NOT_FOUND:
-                              return false;
-                            case BAD_REQUEST:
-                              shouldRetry = false;
-                          }
+                        if (e.getResponseCode() == HttpStatus.BAD_REQUEST.value()) {
+                          shouldRetry = false;
                         }
-                        IntegrationException ie = new IntegrationException(re);
+                        IntegrationException ie = new IntegrationException(e);
```

Comment:
This is potentially a candidate for a followup PR, but I wonder if this exception wrapping is necessary anymore.  Like, we've got a SpinnakerHttpException here, which is already a child of SpinnakerException, and so already has its own retryable attribute.  Plus also, I'd love to depend on SpinnakerRetrofitErrorHandler to set that retryable attribute appropriately.

I don't see any code catching IntegrationException explicitly (except some kork-plugins tests, e.g. [here](https://github.com/spinnaker/kork/blob/b6fa1103b7b3b12734af39cb6f6ca2b04cdd59d6/kork-plugins/src/test/kotlin/com/netflix/spinnaker/kork/plugins/DependencyInjectionTest.kt#L87)), so from that perspective I think the change is OK.

Reply:
Ah, I didn't realize `SpinnakerException` had its own `setRetryable()` method. I can go ahead and use that instead.